media/js/src/assetDetail/AssetDetail.jsx

```diff
@@ -532,37 +532,9 @@ export default class AssetDetail extends React.Component {
                     <div className="col-sm-6">
                         {media}
                         {this.state.tab === 'viewItem' && (
-                            <React.Fragment>
-                                <table className="table mt-1">
-                                    <tbody>
-                                        <tr>
-                                            <th scope="row">Item Name</th>
-                                            <td>
-                                                {this.props.asset.title}
-                                                &nbsp;
-                                                <button type="submit" className="btn btn-secondary btn-sm">Rename</button>
-                                            </td>
-                                        </tr>
-                                        <tr>
-                                            <th scope="row">Permalink</th>
-                                            <td>
-                                                <a href="">
-                                                    {window.location.href}
-                                                </a>
-                                            </td>
-                                        </tr>
-                                        <tr>
-                                            <th scope="row">Creator</th>
-                                            <td>
-                                                {this.props.asset.author.public_name} ({this.props.asset.author.username})
-                                            </td>
-                                        </tr>
-                                    </tbody>
-                                </table>
-                                <button type="submit" className="btn btn-danger btn-sm">
-                                    Delete
-                                </button>
-                            </React.Fragment>
+                            <button type="submit" className="btn btn-danger btn-sm">
```

Comment:
Delete button still appears under the video? That should move up to the Original Item tab as well.